samples/layouts/expansion-panel/properties-and-events/index.html

```diff
@@ -14,7 +14,7 @@
         <div class="panel-wrapper">
           <igc-expansion-panel id="exp-panel">
             <div slot="title">Golden Retriever</div>
-            <div slot="subTitle">Medium-large gun dog</div>
+            <div slot="subtitle">Medium-large gun dog</div>
             <div>
               <p>The Golden Retriever is a medium-large gun dog that retrieves shot waterfowl, such as ducks
               and upland game birds, during hunting and shooting parties.[3] The name "retriever" refers to the breed's ability
```

Comment:
Use `h1` for the title slot. Use `h3` for the subtitle slot. Do not wrap any of the content text in a div or paragraph - simply pass it as a text node.